.github/workflows/dev-release-java-publish-sonatype-and-docker.yml

```diff
@@ -0,0 +1,22 @@
+name: Build and release Java distributions to sonatype.
```

Comment:
Please use manually triggered workflows

Reply:
do we need to publish this component to sonatype?

Reply:
Maven artifacts can be useful when we decide to migrate Pico from Docker image parsing to maven artifacts